benchmark/result_rewriter.py

```diff
@@ -0,0 +1,53 @@
+import json
```

Comment:
Can we see about making a PR to benchmarks to support another output format natively rather than baking it into the security repo?